modules/nf-core/kraken2/kraken2/tests/main.nf.test

```diff
@@ -0,0 +1,145 @@
+nextflow_process {
+    name "Test Process KRAKEN2_KRAKEN2"
+    script "../main.nf"
+    process "KRAKEN2_KRAKEN2"
+    tag "kraken2/kraken2"
+    tag "modules"
+    tag "modules_nfcore"
+
+    setup {
+        run("UNTAR") {
+            script "modules/nf-core/untar/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    [],
+                    file(
+                        params.test_data['sarscov2']['genome']['kraken2_tar_gz'],
+                        checkIfExists: true
+                    )
+                ])
+                """
+            }
+        }
+    }
+
+    test("sarscov2 illumina single end [fastq]") {
+        when {
+            params {
+                outdir   = "$outputDir"
+            }
```

Comment:
no need to set the outdir if asserting on out channels instead of published files in outdir

Reply:
I see, I've removed it now :+1: